src/Widgets/PopoverWidget.vala

```diff
@@ -118,17 +126,16 @@ public class Power.Widgets.PopoverWidget : Gtk.Box {
         }
     }
 
-    public void update_brightness_slider () {
-        screen_brightness.update_slider ();
+    public void update_slider (int new_value) {
+        brightness_slider.val = new_value;
     }
 
```

Comment:
Insert method:
```
public void change_slider (int change) {
    brightness.val += change;
}
```

See comment on Indicator.vala

In fact I think both these methods could be replaced by a `slider_val` property with appropriate getter and setter.